shared/chat/conversation/list.desktop.js

```diff
@@ -114,7 +114,7 @@ class ConversationList extends Component<void, Props, State> {
 
   _invalidateChangedMessages (props: Props) {
     this.state.messages.forEach((item, index) => {
```

Comment:
This is a generalization of the previous check -- before we looked for pending->sent by seeing whether the messageID changed, and now we just look directly at messageState changes, so that we catch pending->sent as well as pending->failed and failed->sent.